modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Enrichments.scala

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2022-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.kinesis
+
+import java.util.Base64
+
+import com.snowplowanalytics.iglu.core.{SchemaKey, SchemaVer}
+
+import com.snowplowanalytics.snowplow.enrich.common.enrichments.registry.YauaaEnrichment
+
+object Enrichments {
```

Comment:
Very nice! I can see JS code now! :D 

Do you think splitting this to enrichment specific objects would make it a bit easier to navigate? I mean sth like: 

* add `enrichments` test package
* put each enrichment in different `object` like `JSEnrichment`, `ApiEnrichment` etc. and define specifics there?